src/CSSTransitionGroup.js

```diff
@@ -53,7 +37,21 @@ class CSSTransitionGroup extends React.Component {
   }
 }
 
-CSSTransitionGroup.propTypes = propTypes;
-CSSTransitionGroup.defaultProps = defaultProps;
+CSSTransitionGroup.propTypes = {
```

Comment:
I could investigate to see whether uglify will find the dead code and resulting file size is same if you'd prefer to keep these as separate variables?